static/css/impala/formset.less

```diff
@@ -32,6 +32,11 @@ input.autocomplete {
         height: 32px;
         margin-right: 4px;
         width: 32px;
+
+        .html-rtl & {
+            float: right;
+            margin-left: 4px;
```

Comment:
I think you should remove the `margin-right` from here too, otherwise looks good! 👍